docs/views/components/NavBar.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navbar = ({ toggleSidebar, isOpen }) => {
+  return (
+    <div className="navbar">
+      <div className="navbar-div">
+        <h1>React Slideshow</h1>
+        <div className="nav-item">
+          <ul>
+            <Link to="/">
+              <li>Docs</li>
+            </Link>
+            <Link to="/api">
+              <li>API</li>
+            </Link>
+            <a
+              target="_blank"
+              href="https://github.com/femioladeji/react-slideshow"
+            >
+              <li>Github</li>
+            </a>
+          </ul>
+        </div>
+        <div className="menu-bar" onMouseDown={toggleSidebar}>
```

Comment:
Why not use an `anchor` tag for this so that you can use the `onClick` event? It's technically a link and for semantics purposes, an `<a>` tag should be used